assets/app/scripts/controllers/deployment.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:PodController
+ * @description
+ * # ProjectController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('DeploymentController', function ($scope, $routeParams, DataService, project, DeploymentsService, ImageStreamResolver, $filter) {
+    $scope.deployment = null;
+    $scope.deploymentConfig = null;
+    $scope.deployments = {};
+    $scope.podTemplates = {};
+    $scope.imageStreams = {};
+    $scope.imagesByDockerReference = {};
+    $scope.imageStreamImageRefByDockerReference = {}; // lets us determine if a particular container's docker image reference belongs to an imageStream
+    $scope.builds = {};     
```

Comment:
yes... unfortunately.  need the image streams to tie the docker image in the container to an image stream, then need the builds to tie a build back to an image stream.  The image stream doesn't have any kind of build annotation so we can't just grab the build we care about.